docs/reference/module-overview.md

```diff
@@ -0,0 +1,284 @@
+
+#### `add`
+
+An object of filters to add to the piece type. Each filter is an object with its own configuration. Filter properties include:
+
+| Property | Description |
+| ------- | ------- |
+| `label` | Recommended, but not required, to customize the human-readable label. |
+| `inputType` | Choose an input field type for the filter from the following: `radio`, `checkbox`, or `select`. Defaults to `select`. |
+| `choices` | Manually set an array of choices instead of letting Apostrophe find the valid choices from the database. Choices include `label` and `value` properties. |
+| `def` | The default value for the manager filter. |
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  filters: {
+    add: {
+      _category: { // 👈 Referencing a relationship field named `_category`
+        label: 'Article category'
+      },
+      featured: { // 👈 Referencing a boolean field name `featured`
+        labeled: 'Featured',
+        inputType: 'checkbox',
+        def: true,
+        choices: [
+          { value: true, label: 'Show featured' },
+          { value: false, label: 'Hide featured' }
+        ]
+      }
+    }
+  }
+};
+```
+
+#### `remove`
+
+An array of filter names from the base class module to remove.
+
+```javascript
+// modules/spotlight-article/index.js
+modules.export = {
+  extend: 'article',
+  filters: {
+    remove: [ 'featured' ]
+  }
+};
+```
+
+### `columns`
+
+In piece type modules, the `columns` setting configures the pieces manager modal, adding and removing the piece listing columns. Default columns include `title`, `updatedAt`, and `visibility`.
+
+#### `add`
+
+An object of columns to add to the piece type manager. Each column is an object with its own configuration. Column properties include:
+
+| Property | Description |
+| ------- | ------- |
+| `label` | Recommended, but not required, to show a column header label. |
+| `component` | An advanced option to use a custom Vue component for table cells in this column. See core components `AposCellBasic` (default) and `AposCellDate` for examples. |
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  extend: '@apostrophecms/piece-type',
+  columns: {
+    // 👇 Adds a column showing when the article was published.
+    add: {
+      lastPublishedAt: {
+        label: 'Published',
+        component: 'AposCellDate'
+      }
+    }
+  }
+};
+```
+
+#### `remove`
+
+An array of column names from the base class module to remove.
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  extend: '@apostrophecms/piece-type',
+  columns: {
+    // 👇 Hides the column showing when the article was last updated.
+    remove: [ 'updatedAt' ]
+  }
+};
+```
+
+### `instantiate`
+
+Set to `false` to prevent the module from being fully instantiated in the application. The primary purpose of this option is to create a base class module that others will [extend](#extend) but that will not be used directly.
```

Comment:
Example? Out of my league.

Reply:
I'm concerned that we're providing too many examples in the reference section already. This is a really deep cut, so I'd rather one day create a recipe/how-to demonstrating a use case. Until then, leave it as "if you don't know what it's for, you probably don't need it."